### DISCUSSION TITLE
Not able to see plugin in Blender

### DISCUSSION THREAD

**Discussion by gioarma** [mitsuba-renderer/mitsuba-blender#76] [Q&A] [answered]
Hello, I have a very basic problem: after installation I'm not able to see the plugin in the Blender Add-on menu, so I'm not able to use the add-on at all.
I followed your installation guide:

- Downloaded the **mitsuba2-blender-v0.1.zip** file at [this page](https://github.com/mitsuba-renderer/mitsuba-blender/releases)
- Opened Blender -> Edit -> Preferences -> Add-ons -> Install -> Select the mitsuba2-blender-v0.1.zip file from the Download folder -> click "Install Add-on". 

The Blender consol prints:
![image](https://github.com/mitsuba-renderer/mitsuba-blender/assets/59478987/a3d158cb-e71e-4719-a64b-1de6aa9dec28)
So everything seems fine. However, when I go back in the Add-on menu and search for "mitsuba" I don't get any available add-on:
![image](https://github.com/mitsuba-renderer/mitsuba-blender/assets/59478987/5eb590fd-d57b-4d1f-aba4-4660d59aeab0)

Looking at the folder `C:\Users\ArmarGio\AppData\Roaming\Blender Foundation\Blender\3.4\scripts\addons\mitsuba2-blender` I see that the add-on was indeed installed:
![image](https://github.com/mitsuba-renderer/mitsuba-blender/assets/59478987/1b8b20b4-db4a-4bae-9921-640845809d44)

## Setup
I am using a Windows machine, with Windows 11 Pro version

I tried using Blender `3.4.1`, and `3.3.0`, but the result was the same. 

I tried this on a different PC, but the result was the same

Any help is appreciated, thanks!

**Comment by bathal1** [comment, marked as answer]
Hi,
The release you tried to download is very outdated. Please download the one called "Latest Build" [here](https://github.com/mitsuba-renderer/mitsuba-blender/releases/download/latest/mitsuba-blender.zip). 

Note that there are some known issues with the exporter on windows for more recent versions of blender (see #61)

**Reply by gioarma** [reply]
Thank you, I was trying to install the old one because I understood the latest build was not stable
